docs/src/getstarted/overview.md

```diff
@@ -16,8 +16,8 @@ keywords:
 
 Welcome to the Solana developer docs!
 
-This guide contains step-by-step instructions on how to get started. Before we
-get into the hands on part of the guide, we'll cover basic concepts that all
+This guide contains step-by-step instructions on how to get started. Before diving
+into the practical aspects of the guide, we'll cover basic concepts that all
```

Comment:
I don't feel that "practical aspects" means the same thing as hands-on. How about adding a hyphen?
```suggestion
This guide contains step-by-step instructions on how to get started. Before
the hands-on part of the guide, we'll cover basic concepts that all
```

Reply:
Thanks! Looks like someone rearranged all the docs under us 😅 I'll take it from here with the rebase.